solidity/contracts/bridge/Bridge.sol

```diff
@@ -405,6 +426,50 @@ contract Bridge is Ownable {
         emit VaultStatusUpdated(vault, isTrusted);
     }
 
+    /// @notice Initiates a new wallet creation.
+    /// @dev ECDSA wallet creation is asynchronous process. Once a wallet is
+    ///      created the ECDSA Wallet Registry will notify this contract by
+    ///      calling a `__ecdsaWalletCreatedCallback` function.
+    function createNewWallet() external {
```

Comment:
I'm also wondering about renaming that function. The name `createNewWallet` suggests a wallet is created within that call and there is an active wallet just after the function call completes. That's not true because this just starts the creation process without any guarantees of success. What do you think about renaming it to, for example, `triggerWalletCreation` or `requestNewWallet`?

Reply:
Totally agree! I like `requestNewWallet`.

Reply:
534e3097aa6a5ed1724a31576c2042096a1e6083